README.md

```diff
@@ -66,6 +67,13 @@ Then run `npm run dev` to compile the documentation. When it finishes, open a ne
 ### Documentation for previous releases
 Previous releases and their documentation are available for [download](https://github.com/materializecss/materialize/releases).
 
+## Build
+If you want to build `materialize.css` or `materialize.js` from the latest commit, you can build the files with the following command after `npm install`. See `package.json` to check the current version like `1.0.0`.
+
+```sh
+npm run release -- --oldver=<current_version> --newver=<new_version>
```

Comment:
I think this all good... well... except the `oldver` argument, it could confuse people who want to build this... maybe just write it `1.0.0` temporary for now instead `<current_version>`?

Reply:
I don't think so. The readme contains an example and is meant for contributors. I think this is sufficient at the moment.

In the future we can use only `newver` or use some package which does this much better and move away from grunt maybe.